### DISCUSSION TITLE
I get error dotenv module not found

### DISCUSSION THREAD

**Discussion by gorillaradio** [imartinez/privateGPT#667] [Q&A]
I followed the instructions but I get a missing dotenv module error. Can anybody help me?

I'm on mac. If I use global Python interpreter (3.10) I get error no dotenv module. If I use conda interpreter I get no langchain module found.

**Comment by osolit** [comment]
rename from example.env

**Reply by gorillaradio** [reply]
I mean a python module named dotenv

**Comment by gorillaradio** [comment]
It works now, I deleted and reinstalled anaconda3 on my mac.
